phoenix-core/src/it/java/org/apache/phoenix/end2end/SortMergeJoinIT.java

```diff
@@ -187,13 +186,13 @@ public void initTable() throws Exception {
                 "CREATE LOCAL INDEX \"idx_supplier\" ON " + JOIN_SUPPLIER_TABLE_FULL_NAME + " (name)"
                 }, {
                 "SORT-MERGE-JOIN (LEFT) TABLES\n" +
-                "    CLIENT PARALLEL 1-WAY RANGE SCAN OVER " + MetaDataUtil.LOCAL_INDEX_TABLE_PREFIX + JOIN_SUPPLIER_TABLE_DISPLAY_NAME + " [-32768]\n" +
+                "    CLIENT PARALLEL 1-WAY RANGE SCAN OVER " +JOIN_SUPPLIER_TABLE_DISPLAY_NAME + " [1]\n" +
```

Comment:
We should update the explain plan code to output something else when a local index is used, otherwise it's too difficult to tell. Is the local index being used?

Something like ... OVER LOCAL INDEX <index name> ...